release/air_tests/air_benchmarks/workloads/tune_benchmark.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+
+import ray
+from ray.train.torch import TorchTrainer
+
+from benchmark_util import time_it
+
+CONFIG = {"lr": 1e-3, "batch_size": 64, "epochs": 20}
+
+
+def prepare_mnist():
+    # Pre-download the data onto each node.
+    from benchmark_util import upload_file_to_all_nodes, run_command_on_all_nodes
+
+    ray.init("auto")
+    print("Preparing Torch benchmark: Downloading MNIST")
+
+    path = os.path.abspath("workloads/_torch_prepare.py")
+    upload_file_to_all_nodes(path)
+    run_command_on_all_nodes(["python", path])
+    ray.shutdown()
+
+
+def get_trainer():
+    """Get the trainer to be used across train and tune to ensure consistency."""
+    from torch_benchmark import train_func
+
+    def train_loop(config):
+        train_func(use_ray=True, config=config)
+
+    trainer = TorchTrainer(
+        train_loop_per_worker=train_loop,
+        train_loop_config=CONFIG,
+        scaling_config={
+            "num_workers": 4,
+            "resources_per_worker": {"CPU": 2},
```

Comment:
I think we'll have to use CPU=4 here to utilize all workers
```suggestion
            "resources_per_worker": {"CPU": 4},
```

Reply:
wait, we are using m5.2xlarge (8CPU) instance and each trial is taking 1 machine. All together there are 8 machines.